### PR TITLE
Proposing a French translation

### PR DESCRIPTION
**Pull request by SphinxKnight**
I hope it will be useful for https://github.com/rust-lang/rust-www/pull/501 :)

### CODE REVIEW

File: fr-FR/community.md [hunk 1]
```diff
@@ -1,101 +1,83 @@
...
+title: La communauté Rust &middot; Rust, le langage de programmation
 ---
 
-# The Rust Community
+# La communauté Rust
 
-The Rust programming language has many qualities, but Rust's greatest
-strength is the community of people who come together to make working
-in Rust a rewarding experience.
+En tant que langage de programmation, Rust possède de nombreuses qualités mais l'un des meilleurs atout de Rust est sa communauté : des personnes qui travaillent ensemble afin que contribuer à Rust soit une expérience enrichissante.
```
Comment (ghost) [inline fr-FR/community.md]: atout > atouts

File: fr-FR/community.md [hunk 2]
```diff
@@ -1,101 +1,83 @@
...
 
-We are committed to providing a friendly, safe and welcoming
-environment for all, regardless of gender, sexual orientation,
-disability, ethnicity, religion, or similar personal
-characteristic. Our [code of conduct][coc] sets the standards for
-behavior in all official Rust forums.
+Nous nous appliquons à fournir un environnement amical, sûr et accueillant pour tous, quel que soit le genre, l'orientation sexuelle, le handicap, l'appartenance ethnique, la religion ou tout aspect personnel. Notre [code de conduite][coc] définit les principes à respecter sur l'eensemble des forums officiels relatifs à Rust.
```
Comment (ghost) [inline fr-FR/community.md]: typo eensemble < ensemble

File: fr-FR/community.md [hunk 3]
```diff
@@ -1,101 +1,83 @@
...
+Les Rustacéens maintiennent différents canaux IRC sur le réseau IRC de Mozilla : irc.mozilla.org.
 
-The [#rust][rust_irc] channel is a venue for general
-discussion about Rust, and a good place to ask for help. You'll find
-people willing to answer any questions about Rust, and
-responses are typically fast.
+Le canal [#rust][rust_irc] est le principal lieu pour les discussions générales à propos de Rust. C'est également un bon endroit pour demander de l'aide. De nombreuses personnes seront prêtes à vous aider et répondront généralement assez rapidement à vos questions.
 
-Rust's developers coordinate in [#rust-internals][internals_irc]. It is for real-time discussion about hacking on Rust itself. It is also the channel to ask questions about contributing to Rust.
+Les développeurs de Rust échangent sur [#rust-internals][internals_irc]. Ce canal est dédié à la fabrication même du langage Rust. C'est également le canal où l'on peut poser des questions sur comment contribuer à Rust.
```
Comment (ghost) [inline fr-FR/community.md]: "sur comment" saymoche. Suggestion "sur la manière de contribuer…"

File: fr-FR/community.md [hunk 4]
```diff
@@ -122,77 +104,53 @@ Rust's developers coordinate in [#rust-internals][internals_irc]. It is for real
...
+Nous avons deux forums pour les échanges asynchrones :
 
-- The [Users Forum][users_forum], a space for asking questions, posting code
-  snippets, talking about Rust projects, and so on.
+- [Le forum des utilisateurs][users_forum], un espace où l'on peut poser des questions, poster des morceaux de code, discuter de projets Rust, etc.
 
-- The [Internals Forum][internals_forum], a space dedicated to design and
-  implementation discussion about Rust itself (which includes Cargo, the
-  standard library, and other core bits of infrastructure).
+- [Le forum interne][internals_forum], un espace destiné aux discusions sur la conception et l'implémentation de Rust même (ce qui inclut Cargo, la bibliothèque standard ainsi que d'autres fragments liés à l'infrastructure).
```
Comment (ghost) [inline fr-FR/community.md]: typo discusions>discussions

File: fr-FR/community.md [hunk 5]
```diff
@@ -204,26 +162,17 @@ help get you started.
 [mod_team]: https://www.rust-lang.org/team.html#Moderation
 [doc_team]: https://www.rust-lang.org/en-US/team.html#Documentation-team
 
-## Rust Development
+## Le développement de Rust
 
-Rust has had over [1,200 different contributors][authors], a number that grows
-every single week. [We'd love for you to join that list][contribute]!
+Rust a plus de [1 200 contributeurs différents][authors] et cela augment chaque semaine. [Nous serions ravis de vous compter parmi nous][contribute] !
```
Comment (ghost) [inline fr-FR/community.md]: typo augment>augmente

**Comment by julienXX** [comment]
👍

**Pull request merged**